ext/random/engine_mt19937.c

```diff
@@ -162,23 +162,7 @@ static uint64_t generate(php_random_status *status)
 
 static zend_long range(php_random_status *status, zend_long min, zend_long max)
```

Comment:
Consider to inline this function.

Reply:
I don't think inlining this is possible, because a function pointer is stored in `php_random_algo_mt19937`:

https://github.com/php/php-src/blob/51a99456e2017d95509bddd668d871cd6ef1a8cb/ext/random/engine_mt19937.c#L246

Anyway with this PR I believe all `range()` implementations are effectively identical, thus possibly allowing the removal of `range()` from `struct php_random_algo` in PHP 8.3 (Internal API breaks are acceptable, right?).

Reply:
> Internal API breaks are acceptable, right?

Is that even an internal API break, or just an ABI break? Anyhow, either is acceptable in a minor version (unless an API break would terribly affect many external extensions).

Reply:
This seems like a very good change. The php_random_algo range is obviously no longer needed.

Is it possible to include the removal of the php_random_algo range in the next RC release? This internal implementation is probably not yet used by any extensions, and if it were, it would be easy to change.

Reply:
> Is that even an internal API break, or just an ABI break?

API, because the struct member would completely disappear, thus breaking all code referencing it.

> Is it possible to include the removal of the php_random_algo range in the next RC release? This internal implementation is probably not yet used by any extensions, and if it were, it would be easy to change.

The next RC is the last RC before the 8.2.0 “gold” release. I don't think cleanup changes should happen now for stability reasons. There's plenty of time to clean this up for PHP 8.3 and as Christoph confirmed, API changes are fine for internals.

Reply:
Indeed. That makes sense. I was a little hasty!